Userland/Applications/PixelPaint/Layer.cpp

```diff
@@ -322,6 +338,53 @@ void Layer::apply_mask()
     delete_mask();
 }
 
+void Layer::invert_mask()
```

Comment:
Hm, i was not aware that this is working... the filter preview does not show the mask - it alwas sticks to the content bitmap - that is probably not intended. But indeed, it can invert the mask :)
While i find the shortcut to invert the mask super handy i can revert that in favor of a quick filter toolbar :) Let's see if the yak herd is going to pick that up ;-)

Reply:
@GMTA While reverting my "Mask Invert" action i noticed that the filter invert does currently not work on Editing-Masks where i would expect that the alpha-values of each pixel are also inverted. Because of that i kept my inverse action for the moment... Whould that be okay for you?

One good thing, i could fix the filter gallery preview to also preview the mask and not always the content bitmap. :-)